xml/System.Net.Http/MultipartContent.xml

```diff
@@ -414,8 +414,8 @@ This operation blocks until all of the content has been written to the memory st
         <ReturnType>System.Net.Http.HeaderEncodingSelector&lt;System.Net.Http.HttpContent&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a callback that encodes response header values.</summary>
+        <value>The header encoding selector callback to decode the value for the specified response header name, or <see langword="null"/> indicating the default behavior.</value>
```

Comment:
Are we encoding or decoding here? In summary you wrote "encode", but in value you wrote "decode", so I am a bit lost

Reply:
it should be `decode`. @MihaZupan can you confirm?

Reply:
@CarnaViire @aik-jahoda due to the time constraint we currently have to get this merged today, I will leave it as it is. If it's wrong, we can get it fixed for the next release.

```suggestion
        <value>The header encoding selector callback to decode the value for the specified response header name, or <see langword="null"/> to indicate the default behavior.</value>
```